app/src/main/java/com/example/fittracker/MainActivityView.java

```diff
@@ -0,0 +1,42 @@
+package com.example.fittracker;
+
+import android.app.Activity;
+import android.widget.EditText;
+import android.widget.Toast;
+
+public class MainActivityView implements LogInContract.View{
+    private EditText mail;
+    private EditText password;
+    private Activity actividad;
+    public MainActivityView(EditText mail, EditText password,Activity actividad){
+        this.mail=mail;
+        this.password=password;
+        this.actividad=actividad;
+    }
+    @Override
+    public String getEmail() {
+        return mail.getText().toString();
+    }
+
+    @Override
+    public String getPassword() {
+        return password.getText().toString();
+    }
+
+    @Override
+    public void logInError() {
+        Toast.makeText(actividad,"Informacion de inicio incorrecta",Toast.LENGTH_SHORT).show();
```

Comment:
if you will keep this toasts the message must be in the resources as a string

Reply:
Done